Readme.md

```diff
@@ -21,3 +21,10 @@ A second difference is that common open source software and specification licens
 ## Who developed the Community Specification
 
 The Community Specification has been developed via the [Joint Development Foundation](http://www.jointdevelopment.org), with inspiration from the [Open Web Foundation agreements](http://openwebfoundation.org) and the [Alliance for Open Media Patent License 1.0](http://aomedia.org/license/patent-license/).
+
+The SPDX Identifier for the Community Specification V1.0 is, [`Community-Spec-1.0`](https://spdx.org/licenses/Community-Spec-1.0.html).
+
+The statement to insert on files to represent this license is:
+
+![APM license](https://img.shields.io/badge/License-Community_Spec-brightgreen)

```

Comment:
A few things here:

* This probably needs to be enclosed in backticks, otherwise the image renders instead of being kept as plaintext
* Was the "APM license" alt-text inadvertently added? I don't think the APM reference applies here...
* I'd probably specifically structure the badge URL to display the correct SPDX identifier

So I think this line would probably instead get the following (with backticks surrounding it for the markdown, and note the double hyphens in the URL):

`![Community-Spec-1.0 license](https://img.shields.io/badge/License-Community--Spec--1.0-brightgreen)`